### PR TITLE
[MIG][13.0] base_territory

### PR DESCRIPTION
**Pull request by murtuzasaleh**

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
#373

File: base_territory/models/res_country.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCountry(models.Model):
+    _inherit = "res.country"
+
+    territory_id = fields.Many2one("res.territory", string="Territory")
```
Comment (max3903) [inline base_territory/models/res_country.py:10]: @murtuzasaleh A country has many regions.

CC: @wolfhall

File: base_territory/models/res_state.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCountryState(models.Model):
+    _inherit = "res.country.state"
+
+    territory_id = fields.Many2one("res.territory", string="Territory")
```
Comment (max3903) [inline base_territory/models/res_state.py]: @murtuzasaleh Not needed. Can be removed.

Comment (murtuzasaleh) [reply base_territory/models/res_state.py]: @max3903 Can I remove also `state_ids` One2many fields from `res_territory.py` file? https://github.com/OCA/field-service/pull/508/files#diff-859251c00fcc1e8c6bdeb87872dfcad7R22

Comment (max3903) [reply base_territory/models/res_state.py]: @murtuzasaleh Yes.

**Review by max3903 — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]
👍

**Pull request merged**